src/main/resources/network/brightspots/rcv/hints_winning_rules.txt

```diff
@@ -22,6 +22,8 @@ Use Batch Elimination: Batch elimination, or simultaneous elimination of all can
 
 Continue until Two Candidates Remain: Single-winner ranked-choice voting elections can stop as soon as a candidate receives a majority of votes, even though 3 or more candidates may still be in the race. Selecting this option will run the round-by-round count until only two candidates remain, regardless of when a candidate wins a majority of votes. Available only when Winner Election Mode is "Single-winner majority determines winner" or "Multi-pass IRV."
 
+Stop Tabulation Early: If a winner is not found by the given round, tabulation stops early.
```

Comment:
Does anybody else need to look over this to make sure it's sensible? Does this sync with any external documentation?

Reply:
Approved by Rene